packages/eslint-plugin/tests/rules/unbound-method.test.ts

```diff
@@ -43,6 +44,14 @@ function addContainsMethodsClassInvalid(
 
 ruleTester.run('unbound-method', rule, {
   valid: [
+    'Promise.resolve().then(console.log)',
+    '["1", "2", "3"].map(Number.parseInt)',
+    '[5.2, 7.1, 3.6].map(Math.floor);',
+    'const x = console.log',
+    `
+import console from './consoleshim';
+Promise.resolve().then(console.log);
+      `,
```

Comment:
probs should remove this.
This only passes because typescript cannot resolve the import.